README.md

```diff
@@ -75,6 +75,17 @@ executed directly inside the `policy` module.
 mvn clean generate-sources -DskipPolicyManClientGeneration=false
 ```
 
+
```

Comment:
we can remove this change in the readme file. I will update the documentation on the website with userstory https://github.com/eclipse-xpanse/xpanse/issues/2355